src/include/btmem.h

```diff
@@ -128,6 +128,46 @@ __wt_page_header_byteswap(WT_PAGE_HEADER *dsk)
 #define WT_PAGE_HEADER_BYTE(btree, dsk) \
     ((void *)((uint8_t *)(dsk) + WT_PAGE_HEADER_BYTE_SIZE(btree)))
 
+/*
+ * WT_DELTA_HEADER --
+ *	Header for delta
+ */
```

Comment:
I assume this is in progress, as this struct is not yet used....?

Reply:
yes, it is in progress.

Reply:
[3dded8e](https://github.com/wiredtiger/wiredtiger/pull/10959/commits/3dded8ec86487961cde0839f14464dfc28b0b632)